client/src/components/PageLayout/Header.tsx

```diff
@@ -1,78 +1,43 @@
 import { HStack } from '@chakra-ui/layout';
-import { Box, Flex, Image, Spinner, MenuItem } from '@chakra-ui/react';
-import type { GridItemProps } from '@chakra-ui/react';
+import {
+  Box,
+  Image,
+  Button,
+  Flex,
+  Menu,
+  MenuList,
+  MenuItem,
+  MenuButton,
+} from '@chakra-ui/react';
 import { Link } from 'chakra-next-link';
 import { SkipNavLink } from '@chakra-ui/skip-nav';
 import { useRouter } from 'next/router';
-import React, { forwardRef } from 'react';
-import NextLink from 'next/link';
+import React from 'react';
 
-import { useAuth } from '../../modules/auth/store';
-import styles from '../../styles/Header.module.css';
+import NextLink from 'next/link';
 import Avatar from '../Avatar';
-import { Permission } from '../../../../common/permissions';
-import { checkPermission } from '../../util/check-permission';
-import { HeaderMenu } from './HeaderMenu';
+import styles from '../../styles/Header.module.css';
+import { useAuth } from '../../modules/auth/store';
+import { HeaderContainer } from './component/HeaderContainer';
 import { useLogout, useLogin } from 'hooks/useAuth';
-import { MeQuery } from 'generated/graphql';
-
-interface Props {
-  children: React.ReactNode;
-  justifyContent?: GridItemProps['justifyContent'];
-}
-
-const HeaderItem = forwardRef<HTMLDivElement, Props>((props, ref) => {
-  return (
-    <Flex
-      justifyContent="space-between"
-      alignItems="center"
-      ref={ref}
-      {...props}
-      w="full"
-      as="header"
-      height={'4.5em'}
-      px={[2, 4, 8]}
-      py={[2, 4]}
-      background={'gray.85'}
-      className={styles.header}
-    />
-  );
-});
 
 // TODO: distinguish between logging into the app and logging into Auth0. Maybe
 // use sign-in for the app?
-const LoginButton = () => {
-  const login = useLogin();
-  return (
-    <MenuItem
-      onClick={login}
-      fontWeight="600"
-      background={'gray.85'}
-      color={'gray.10'}
-      height={'100%'}
-      borderRadius={'5px'}
-      _hover={{ color: 'gray.85' }}
-    >
-      Log In
-    </MenuItem>
-  );
-};
 
 export const Header: React.FC = () => {
   const router = useRouter();
-  const { user, loadingUser } = useAuth();
+  const { user } = useAuth();
```

Comment:
Why were we using `loadingUser`? I remember approving its PR, but I can't remember/find the PR

Reply:
It's just there so we can show a spinner while we're fetching user data.

Reply:
Adding it back 👍, should I add comment about it?

Reply:
Sure, if it's not clear from the context, then a comment's fine.